src/main/java/uk/gov/ida/matchingserviceadapter/validators/validationrules/ConditionsElementMustNotBeNull.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.ida.matchingserviceadapter.validators.validationrules;
+
+import org.opensaml.saml.saml2.core.Conditions;
+import uk.gov.ida.matchingserviceadapter.exceptions.SamlResponseValidationException;
+
+public class ConditionsElementMustNotBeNull extends ValidationRule<Conditions> {
+
+    public ConditionsElementMustNotBeNull() {
+        super((e) -> e != null);
```

Comment:
Should this be `(e) -> e == null`? It looks like `ValidationRule` will throw if the result of this is false?